### ISSUE TITLE
Cookie bar nor working on https://cookie-bar.eu/

### ISSUE THREAD

**Issue by Felwin** [ToX82/cookie-bar#141]
I tried with Chrome, Firefox and Edge on Windows and Chrome on Android and not a single time was I able to see an actual cookie bar. 
I believe there should be one that we customize ?

Clicked the different buttons on the page, including "Click here to revoke the Cookie consent"  and "Update demo" 
Tried incognito mode, deactivate plugin etc.

**Comment by ToX82** [comment]
Hello @Felwin ,
I'm actually having the same issue with Vivaldi browser, but I can see the cookie bar on Brave, chromium, firefox, opera.

I would assume that there are some issues on your side or with some specific browsers, but I'm not totally sure. Could you check on your developer tools -> console if you see any errors like this? `Failed to load resource: net::ERR_BLOCKED_BY_CLIENT`

**Comment by Felwin** [comment]
No, if I clear my cookies I see some square at the bottom right instead of the chat saying i need to accept the cookie to get help, but no cookie consent message.
When I reload the page I can see the chat and I have a cookie saying I gave my consent.
